integrations/docker/images/vscode/chip-build-vscode/Dockerfile

```diff
@@ -17,6 +17,7 @@ FROM ghcr.io/project-chip/chip-build-ti:${VERSION} AS ti
 FROM ghcr.io/project-chip/chip-build-openiotsdk:${VERSION} AS openiotsdk
 FROM ghcr.io/project-chip/chip-build-bouffalolab:${VERSION} AS bouffalolab
 FROM ghcr.io/project-chip/chip-build-asr:${VERSION} AS asr
+FROM ghcr.io/project-chip/chip-build-nuttx:${VERSION} AS nuttx
```

Comment:
something has to be copied from nuttx ... I expect some compiler and sdk?

Reply:
The sdk is dynamically downloaded at compile time to facilitate the upgrade of the NuttX version, is it acceptable?

Reply:
Generally no - we need reproducible builds and we want to be able to lock versions. This way if we need to go back and investigate some issue/bisect chantes we can go back and check what caused them.

Reply:
@andy31415 OK, I also download the NuttX SDK into chip-build-nuttx and copy /opt/nuttx into chip-build-vscode. Is that okay?